module/learning-resources/validator/v1.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * name : v1.js
+ * author : Rakesh
+ * created-date : 28-Nov-2020
+ * Description : Learning Resources.
+ */
+module.exports = (req) => {
+
+    let laearningResourcesValidator = {
+
+        list: function () {
```

Comment:
@rakeshSgr - No validations required here? If not remove the file itself

Reply:
its not require, I have removed the file